whitelabelco/sofo-foods
# Cp/feat/contact form

### Description 💻

Resolves: #91 

### Implementation 💡

creates a responsive contact form section  - form submit does not post anywhere but data can be confirmed, still need api key from client
[Comps](https://www.figma.com/design/04vc8B8NFKlryNS0e1h57T/SoFo-Foods?node-id=168-2353&node-type=frame&t=dcfbxeT0R0ix3W0M-0)

#### Type of change


### How to test 🔍

- `npm i`
- `npm run dev`
- navigate to `http://localhost:3000/`
- click on "Become a Customer" button
- compare to comps and test on responsive screens 
- fill out form and confirm data in console logs

### Screenshots/Demos 📷


### Checklist ✅
